service/dap/server_test.go

```diff
@@ -216,6 +217,133 @@ func TestLaunchTestRequest(t *testing.T) {
 	})
 }
 
+// Test requests that are not supported and return empty responses.
+func TestNoopResponses(t *testing.T) {
+	var got, want dap.Message
+	runTest(t, "increment", func(client *daptest.Client, fixture protest.Fixture) {
+		newResponse := func(reqSeq int, command string) dap.Response {
+			return dap.Response{
+				ProtocolMessage: dap.ProtocolMessage{Type: "response"},
+				RequestSeq:      reqSeq,
+				Success:         true,
+				Command:         command,
+			}
+		}
+
+		client.InitializeRequest()
+		client.ExpectInitializeResponse(t)
+
+		client.SetExceptionBreakpointsRequest()
+		got = client.ExpectSetExceptionBreakpointsResponse(t)
+		want = &dap.SetExceptionBreakpointsResponse{Response: newResponse(1, "setExceptionBreakpoints")}
+		if !reflect.DeepEqual(got, want) {
+			t.Errorf("\ngot  %#v\nwant %#v", got, want)
+		}
+		client.TerminateRequest()
+		got = client.ExpectTerminateResponse(t)
+		want = &dap.TerminateResponse{Response: newResponse(2, "terminate")}
+		if !reflect.DeepEqual(got, want) {
+			t.Errorf("\ngot  %#v\nwant %#v", got, want)
+		}
+		client.RestartRequest()
+		got = client.ExpectRestartResponse(t)
+		want = &dap.RestartResponse{Response: newResponse(3, "restart")}
+		if !reflect.DeepEqual(got, want) {
+			t.Errorf("\ngot  %#v\nwant %#v", got, want)
+		}
+		client.SetFunctionBreakpointsRequest()
```

Comment:
Nit: add an empty line between each request test (after the closing `}` of the comparison `if`, to make them a little easier to distinguish visually

Reply:
Done